Makefile

```diff
@@ -26,16 +28,34 @@ container: clean
 
 
 runcontainer:
-	$(docker) run --rm --name $(name) --env-file dev.env --log-opt max-size=50m -ti -p 4848:4848 $(image)
-
+	@if [ "$(persist)" = "true" ]; then \
+		$(docker) run --rm --name $(name) -v ./container_config:/config -v ./container_downloads:/downloads -v ./tubesync/sync/management/commands:/app/sync/management/commands --env-file dev.env --log-opt max-size=50m -ti -p 4848:4848 -d $(image); \
+		$(MAKE) container_logs;\
+	else \
+		$(docker) run --rm --name $(name) --env-file dev.env --log-opt max-size=50m -ti -p 4848:4848 $(image);\
+	fi
 
 stopcontainer:
-	$(docker) stop $(name)
+	-$(docker) stop $(name)
+
+container_logs:
+	$(docker) logs -f $(name)
 
+container_exec:
+	$(docker) exec -ti $(name) $(exec)
+
+container_manage:
+	$(docker) exec -ti $(name) python3 manage.py $(command)
```

Comment:
Similarly to `container_exec` above.
Where is the value for `$(command)` coming from?